### ISSUE TITLE
SPC a d is undefined with vinegar layer enabled

### ISSUE THREAD

**Issue by HEMCSEC** [syl20bnr/spacemacs#5709]
SPC a d is undefined with vinegar layer enabled

#### Description
DESCRIPTION OF THE PROBLEM

#### Reproduction guide
- Start Emacs
- Enable vinegar layer
- SPC a d; SPC f j

*Observed behaviour:*
SPC a d is undefined; SPC f j is undefined

*Expected behaviour:*
DESCRIPTION OF THE EXPECTED BEHAVIOUR

#### System Info
- OS: darwin
- Emacs: 25.0.92.1
- Spacemacs: 0.105.16
- Spacemacs branch: develop (rev. 64392e3)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-snippets-in-popup t auto-completion-enable-help-tooltip t)
 better-defaults chinese
 (c-c   :variables c-c  -default-mode-for-headers 'c  -mode c-c  -enable-clang-support t)
 django emacs-lisp gtags git html javascript markdown org osx
 (latex :variables latex-build-command "LatexMk")
 python
 (shell :variables shell-default-height 30 shell-default-position 'bottom)
 spacemacs-helm semantic
 (spell-checking :variables spell-checking-enable-by-default nil)
 (syntax-checking :variables syntax-checking-enable-by-default nil)
 vimscript
 (vinegar :variables vinegar-reuse-dired-buffer t))
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

**Comment by HEMCSEC** [comment]
When I change (defun vinegar/init-dired () in packages.el of layer vinegar to (defun vinegar/post-init-dired (), SPC a d works again.

**Comment by StreakyCobra** [comment]
Indeed, we fixed the same problem for ranger recently. The solution you proposed is the right way to go, you can PR it.

**Comment by StreakyCobra** [comment]
This has been fixed in 0dc19d127b189625d748fa5f74ed9a86a6acc1a3. If it is working for you now you can close the issue :-)

**Comment by HEMCSEC** [comment]
It works, thank you.
